src/cnxml/document.js

```diff
@@ -155,18 +175,46 @@ export const IMAGE = block('image', de_image, 'image', se_image)
  * Process data for audio.
  */
 function de_audio(el) {
+    const data = {
+        mime: el.getAttribute('mime-type'),
+        src: el.getAttribute('src'),
+    }
+    if (el.hasAttribute('autoplay')) {
+        data.autoplay = el.getAttribute('autoplay') === 'true'
+    }
+    if (el.hasAttribute('controller')) {
+        data.controller = el.getAttribute('controller') === 'true'
+    }
+    if (el.hasAttribute('for')) {
+        data.for = el.getAttribute('for')
+    }
+    if (el.hasAttribute('longdesc')) {
+        data.longdesc = el.getAttribute('longdesc')
+    }
+    if (el.hasAttribute('loop')) {
+        data.autoplay = el.getAttribute('loop') === 'true'
+    }
+    if (el.hasAttribute('standby')) {
+        data.standby = el.getAttribute('standby')
+    }
+    if (el.hasAttribute('volume')) {
+        data.volume = Number(el.getAttribute('volume'))
+    }
```

Comment:
I'm not convinced we want to support all those attributes (except for `for` but I could not not select it). We should first find examples in OpenStax's books (preferably, third party books on CNX would be find) of how those attributes are used and how they are transformed into HTML. Also see discussion of longdesc above.